metrics/report/mail.go

```diff
@@ -0,0 +1,87 @@
+/// Copyright (c) 2017 Intel Corporation
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package main
+
+import (
+	"log"
+	"net/smtp"
+	"strings"
+)
+
+// bodyMsg concatenates the output of
+// checktool in one string, this string
+// is the body of the email message.
+func bodyMsg(outRows []string) string {
+	var body string
+
+	body += "<h3>PnP Checktool results</h3>\n"
+
+	for _, row := range outRows {
+		body += row + "<br>"
```

Comment:
Sure, but it's an opening tag and there is no corresponding closing tag, so it will generate invalid xhtml/XML.

You can fix it by changing it to `<br/>`.